src/components/basic/input/text-input/PPTextInput.tsx

```diff
@@ -0,0 +1,60 @@
+import clsx from "clsx";
```

Comment:
whats this

Reply:
It's a react package which give you a bit more freedom when adding conditional classes.

https://www.npmjs.com/package/clsx